api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -159,4 +159,68 @@ protected function doFromObject($srcObj, KalturaDetachedResponseProfile $respons
 		parent::doFromObject($srcObj, $responseProfile);
 	}
 
+	public function validateForInsert($propertiesToSkip = array())
+	{
+		parent::validateForInsert($propertiesToSkip);
+		$this->validatePropertyNotNull("entryId");
+		$this->validateEntryId();
+		$this->validateUserID();
+	}
+	
+	public function validateForUpdate($sourceObject, $propertiesToSkip = array())
+	{
+		if($this->entryId !== null)
+			$this->validateEntryId($sourceObject->getId());
+
+		$this->validateUserID($sourceObject->getId());
+		return parent::validateForUpdate($sourceObject, $propertiesToSkip);
+	}
+	
+	/*
+	 * @param string $userEntryID
+	 * @throw KalturaAPIException
+	 */
+	private function validateEntryId($userEntryID = null)
```

Comment:
No need for this method to be private - we may want to override on KalturaUserEntry child classes, better make if protected.